_topic_maps/_topic_map.yml

```diff
@@ -3576,19 +3576,24 @@ Topics:
   Topics:
   - Name: Distributed tracing architecture
```

Comment:
This could be in the same level as [Distributed tracing release notes](https://61948--docspreview.netlify.app/openshift-enterprise/latest/distr_tracing/distributed-tracing-release-notes.html) Currently we have to click on the Distributed tracing architecture dropdown arrow and then again select Distributed tracing architecture to view the page.

Reply:
agree, but I consider this out of the scope of this work.  I will open a separate PR to fix this